test/driver.js

```diff
@@ -673,3 +673,5 @@ var Driver = (function DriverClosure() {
 
   return Driver;
 })();
+
+exports.Driver = Driver;
```

Comment:
I don't really understand this change. There is no `exports` variable in this file. Could you explain this a bit?